<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Joerg Wunsch &lt;j@uriah.heep.sax.de&gt;</author>
 <class>IDlgTrackFilter</class>
 <widget class="QDialog" name="IDlgTrackFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track Filter...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabTrackFilter">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDataset">
      <attribute name="title">
       <string>Filter Track</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="labelDataset">
             <property name="text">
              <string>Hide track points if:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioDistance">
             <property name="text">
              <string>Distance to previous point less than</string>
             </property>
             <attribute name="buttonGroup">
              <string>buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinDistance">
             <property name="suffix">
              <string notr="true">m</string>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="comboMeterFeet">
             <item>
              <property name="text">
               <string notr="true">m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">ft</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="spinTimedelta">
             <property name="suffix">
              <string notr="true">s</string>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QRadioButton" name="radioTimedelta">
             <property name="text">
              <string>Time from previous point less than</string>
             </property>
             <attribute name="buttonGroup">
              <string>buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spinAzimuthDelta">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string>Â°</string>
             </property>
             <property name="maximum">
              <number>90</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="Line" name="line">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkAzimuthDelta">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Azimuth from previous point less than</string>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Change track point data:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="3">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QCheckBox" name="checkMedian">
             <property name="text">
              <string>Smooth profile (Median filter, 5 tabs)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>If you select to split the track, the above will apply to the newly created fragments.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>147</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkReduceDataset">
         <property name="text">
          <string>Perform dataset reduction on &quot;OK&quot;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTimestamp">
      <attribute name="title">
       <string>Timestamps</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="labelStartTime">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Change the track's starting time (anonymize track) to:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QDateTimeEdit" name="datetimeStartTime">
             <property name="displayFormat">
              <string notr="true">yyyy-MM-dd HH:mm:ss</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="radioLocalTime">
               <property name="text">
                <string>Local time</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string>buttonGroupTZ</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioUTC">
               <property name="text">
                <string>UTC</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string>buttonGroupTZ</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0" colspan="2">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="buttonReset1stOfMonth">
               <property name="text">
                <string>Reset to 1st of month, hour 00</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="buttonResetEpoch">
               <property name="text">
                <string>Reset to 1970-01-01, 00:00 UTC</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="spacer1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="radioDeltaOriginal">
         <property name="text">
          <string>Leave time deltas as in original track</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroupTDelta</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioDelta1s">
         <property name="text">
          <string>Change time deltas to 1 second per trackpoint</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroupTDelta</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>If you select to split the track, the above will apply to the newly created fragments.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>381</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkModifyTimestamps">
         <property name="text">
          <string>Modify timestamps on &quot;OK&quot;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Split Track</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioSplitChunks">
           <property name="text">
            <string>Split the track into </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinSplitChunks">
           <property name="minimum">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>chunks</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioSplitPoints">
           <property name="text">
            <string>SplitTrack into chunks of </string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinSplitPoints">
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>points</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSplitTrack">
         <property name="text">
          <string>Split track on &quot;Ok&quot;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabTrackFilter</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>radioDistance</tabstop>
  <tabstop>spinDistance</tabstop>
  <tabstop>comboMeterFeet</tabstop>
  <tabstop>checkAzimuthDelta</tabstop>
  <tabstop>spinAzimuthDelta</tabstop>
  <tabstop>radioTimedelta</tabstop>
  <tabstop>spinTimedelta</tabstop>
  <tabstop>checkReduceDataset</tabstop>
  <tabstop>radioUTC</tabstop>
  <tabstop>buttonReset1stOfMonth</tabstop>
  <tabstop>buttonResetEpoch</tabstop>
  <tabstop>radioDeltaOriginal</tabstop>
  <tabstop>radioDelta1s</tabstop>
  <tabstop>checkModifyTimestamps</tabstop>
  <tabstop>radioSplitChunks</tabstop>
  <tabstop>spinSplitChunks</tabstop>
  <tabstop>radioSplitPoints</tabstop>
  <tabstop>spinSplitPoints</tabstop>
  <tabstop>checkSplitTrack</tabstop>
  <tabstop>datetimeStartTime</tabstop>
  <tabstop>radioLocalTime</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IDlgTrackFilter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IDlgTrackFilter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupTDelta"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroupTZ"/>
 </buttongroups>
</ui>
